source/usb/hid/usbd_hid.c

```diff
@@ -32,6 +32,7 @@ BOOL DataOutAsyncReq;
 U32 DataOutUpdateReqMask;
 U8 *ptrDataOut;
 volatile U16 DataOutToSendLen;
+volatile BOOL LastWasWebusb;
```

Comment:
Flag to switch between HID interface (0x03) and new interface (0xFF) used by WebUSB